packages/components/tests/dummy/app/templates/components/page-header.hbs

```diff
@@ -0,0 +1,358 @@
+
+      <Hds::PageHeader as |PH|>
+        <PH.Title>Page title</PH.Title>
+        <PH.Generic>
+          <Shw::Placeholder @text="generic content" @width="500" @height="36" @background="#e1f5fe" />
+        </PH.Generic>
+      </Hds::PageHeader>
+
+    </SF.Item>
+
+  </Shw::Flex>
+
+  <Shw::Divider @level={{3}} />
+
+  <Shw::Text::H2>Complex examples</Shw::Text::H2>
+
+  <Shw::Flex @direction="column" as |SF|>
+
+    <SF.Item @label="Product landing page">
+
+      <Hds::PageHeader as |PH|>
+        <PH.Title>HCP Packer</PH.Title>
+        <PH.Breadcrumb>
+          <Hds::Breadcrumb>
+            <Hds::Breadcrumb::Item @text="Sandbox" @icon="org" />
+            <Hds::Breadcrumb::Item @text="HCP Packer" @icon="packer" />
+            <Hds::Breadcrumb::Item @text="jane-doe" @icon="file-text" />
+          </Hds::Breadcrumb>
+        </PH.Breadcrumb>
+        <PH.IconTile @icon="packer-color" @color="packer" />
+        <PH.Description>Create and view Packer registries in HCP.
+        </PH.Description>
+        <PH.Actions>
+          <Hds::Button @text="Create Packer registery" @icon="plus" @iconPosition="leading" />
+        </PH.Actions>
+      </Hds::PageHeader>
+
+      <Shw::Divider @level={{2}} />
+
+    </SF.Item>
+
+    <SF.Item @label="Item page">
+
+      <Hds::PageHeader as |PH|>
+        <PH.Title>Boundary clusters</PH.Title>
+        <PH.IconTile @icon="boundary-color" @color="boundary" />
+        <PH.Breadcrumb>
+          <Hds::Breadcrumb>
+            <Hds::Breadcrumb::Item @text="Cloud sandbox" @icon="org" />
+            <Hds::Breadcrumb::Item @text="Boundary clusters" @icon="boundary" />
+          </Hds::Breadcrumb>
+        </PH.Breadcrumb>
+        <PH.Badges>
+          <Hds::Badge @text="Active" @icon="check" @color="success" />
+        </PH.Badges>
+        <PH.Subtitle>boundary-cluster-name</PH.Subtitle>
+      </Hds::PageHeader>
+
+      <Shw::Divider @level={{2}} />
+
+    </SF.Item>
+
+    <SF.Item @label="Multiple badges">
+
+      <Hds::PageHeader as |PH|>
+        <PH.Title>Consul cluster</PH.Title>
+        <PH.Badges>
+          <Hds::Badge @text="Active" @icon="check" @color="success" />
+          <Hds::Badge @text="Deployed to AWS west-2" @color="highlight" @icon="rocket" />
+          <Hds::BadgeCount @text="v1.5" @color="neutral" />
+          <Hds::Badge @text="Locked by jane-doe" @color="warning" @icon="lock" />
+        </PH.Badges>
+        <PH.Description>
+          While this is supported by the contextual component, this many badges being used to represent metadata can be
+          problemantic.
+        </PH.Description>
+      </Hds::PageHeader>
+
+      <Shw::Divider @level={{2}} />
+
+    </SF.Item>
+
+    <SF.Item @label="Kitchen sink example">
+
+      <Hds::PageHeader as |PH|>
+        <PH.Title>Users</PH.Title>
+        <PH.Subtitle>Access control (IAM)</PH.Subtitle>
+        <PH.Description>Manage organization users, roles, and access control.
+        </PH.Description>
```

Comment:
```suggestion
        <PH.Description>Manage organization users, roles, and access control.</PH.Description>
        <PH.Description>And this is a second description, in case multiple paragraphs are needed.</PH.Description>
```